plugins/standard/applets/conference/conferences.js

```diff
@@ -0,0 +1,62 @@
+function join_call(conference_name){
+		
+	window.parent.Client.call({ 
+		'conference_name' : conference_name,
+		'muted' : 'false',
+		'beep' : 'true',
+		'Digits' : 1});
+	
+	return false;
+}
+
+function listen_call(conference_name){
+
+	window.parent.Client.call({ 
+		'conference_name' : conference_name,
+		'muted' : 'true',
+		'beep' : 'false',
+		'Digits' : 1});
+	
+	return false;
+}
+
+$(document).ready(function(){
+	jQuery(function($) {
+		$(function(){
+					
+			var
+			calls = {},
+			select = $('#calls'),
+			updateCalls = function() {
+				$.getJSON(window.location + '/?json=1', function(data) {
+					$.each(data, function(conference_name, call) {
+						if(!calls[conference_name]) {
+							calls[conference_name] = call;
+							
+							row = '<tr class="items-row" id="' + call.friendly_name + '">';
+							row += '<td>' + call.date_created + '</td>';
+							row += '<td>' + call.friendly_name + '</td>';
+							row += '<td>' + call.status + '</td>';
+							row += '<td><a onclick="join_call(\'' + call.friendly_name + '\');">Join</a></td>';
```

Comment:
I'd still like all JS actions to be handled via listeners. We only really use `onclick` for `window.open` and I'd even like to go through and nuke all of those too.